apps/nrm/behaviours/format-answers.js

```diff
@@ -0,0 +1,129 @@
+ * @param {Object} text - text to be capitalised
+ * @param {bool} [all=false] - whether the first letter of each word is capitalised
+ *
+ * @returns {bool} - capitalised text
+ */
+const capitaliseText = (text, all = false) => {
+  const capitalisedText = text.charAt(0).toUpperCase() + text.slice(1);
+
+  if (all) {
+    var splitString = text.toLowerCase().split(' ');
+    for (var i = 0; i < splitString.length; i++) {
+      splitString[i] = splitString[i].charAt(0).toUpperCase() + splitString[i].substring(1);
+    }
+    return splitString.join(' ');
+  }
+
+  return capitalisedText;
+
+};
+
+/**
+ * Remove dashes from text
+ *
+ * @param {Object} text - request parameters
+ *
+ * @returns {bool} - has the user followed the duty to notify path
+ */
+const removeDashesFromText = (text) => {
+  const strippedDashes = text.replace('-', ' ');
+
+  return strippedDashes;
+};
+
+/**
+ * Format session answers
+ *
+ * @param {Object} req - request object
+ *
+ * @returns {void}
+ */
+const formatAnswers = (req) => {
+  req.sessionModel.set(
+    'pv-under-age',
+    removeDashesFromText(capitaliseText(req.sessionModel.get('pv-under-age'))),
+  );
+
+  if (req.sessionModel.get('does-pv-need-support')) {
+    req.sessionModel.set(
+      'does-pv-need-support',
+      capitaliseText(req.sessionModel.get('does-pv-need-support')),
+    );
+  }
+
+  if (req.sessionModel.get('pv-want-to-submit-nrm')) {
+    req.sessionModel.set(
+      'pv-want-to-submit-nrm',
+      capitaliseText(req.sessionModel.get('pv-want-to-submit-nrm')),
+    );
+  }
+
+  if (req.sessionModel.get('co-operate-with-police')) {
+    req.sessionModel.set(
+      'co-operate-with-police',
+      capitaliseText(req.sessionModel.get('co-operate-with-police')),
+    );
+  }
+
+  if (req.sessionModel.get('pv-gender')) {
+    req.sessionModel.set(
+      'pv-gender',
+      removeDashesFromText(capitaliseText(req.sessionModel.get('pv-gender'))),
+    );
+  }
+
+  if (req.sessionModel.get('does-pv-have-children')) {
+    req.sessionModel.set(
+      'does-pv-have-children',
+      capitaliseText(req.sessionModel.get('does-pv-have-children')),
+    );
+  }
+
+  if (req.sessionModel.get('pv-interpreter-requirements')) {
+    req.sessionModel.set(
+      'pv-interpreter-requirements',
+      capitaliseText(req.sessionModel.get('pv-interpreter-requirements')),
+    );
+  }
+
+  if (req.sessionModel.get('pv-other-help-with-communication')) {
+    req.sessionModel.set(
+      'pv-other-help-with-communication',
+      capitaliseText(req.sessionModel.get('pv-other-help-with-communication')),
+    );
+  }
+
+  if (req.sessionModel.get('pv-under-age-at-time-of-exploitation')) {
+    req.sessionModel.set(
+      'pv-under-age-at-time-of-exploitation',
+      removeDashesFromText(capitaliseText(req.sessionModel.get('pv-under-age-at-time-of-exploitation'))),
+    );
+  }
+
+  req.sessionModel.set(
+    'fr-location',
+    removeDashesFromText(capitaliseText(req.sessionModel.get('fr-location'), true)),
```

Comment:
hmm 🤔 not sure about having spaces as a value in the session, can you not just add it in Locals and just display it, see how it's done in the PDF